client/components/file-picker/docs/example.jsx

```diff
@@ -48,7 +48,7 @@ export default class FilePickers extends Component {
 
 				<h4>Any internal content works:</h4>
 				<FilePicker onPick={ this.onSingle }>
-					<a href="#">Select File…</a>
+					<a href="#input">Select File…</a>
```

Comment:
Could you explain this rule fix a little more?

Reply:
The rule that was highlighted here is:

```
The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md
```

Now, typically I would refer this to the team that owns this component for the fix, but since this is an example file, I felt it was okay to add a href value.

Reply:
Ah sure, that makes sense! Linking to an element with an ID of `input` just felt like a kind of strange/uncommon application

Per that rule, it sounds like this element might be better suited as a button? It's really just there to trigger an event!

Anyway, that's a tiny nit and not required! 😄